test-schema.json

```
@@ -27,6 +27,20 @@
                 "type": "array",
                 "items": { "$ref": "#/$defs/test" },
                 "minItems": 1
+            },
+            "specification":{
+                "description": "Location of the test case in the specification",
```

It's not just "the" specification, given it can either be the JSON Schema specification or some other specification. Probably we should elaborate here with some guidance for how to use this, specifically:

```suggestion
                "description": "A reference to a specification document which defines the behavior tested by this test case. Typically this should be a JSON Schema specification document, though in cases where the JSON Schema specification points to another RFC it should contain *both* the portion of the JSON Schema specification which indicates what RFC (and section) to follow as *well* as information on where in that specification the behavior is specified.",
```

---

Got this 👍